plugins.d/python.d.plugin

```diff
@@ -515,6 +519,9 @@ def parse_cmdline(directory, *commands):
             mods.append(cmd.replace(".chart.py", ""))
         else:
             try:
+                # FIXME for some reason this overwrites the module configuration
```

Comment:
This might not be right. By default, python modules configuration should not have an `update_every`. So, the environment variable `NETDATA_UPDATE_EVERY` and/or the command line option should set the default.